frontend/wallet/package.json

```diff
@@ -31,8 +32,35 @@
     "query-fake": "concurrently --kill-others 'npm run fake' 'sleep 5 && npm run query'",
     "reformat": "bsrefmt --in-place src/*.re",
     "test": "jest",
+    "dist": "build",
     "dev": "concurrently --kill-others 'npm run fake' 'bsb -make-world -w' 'fpack watch --development ./lib/js/src/Index.js' 'sleep 2 && electron ./lib/js/src/App.js'"
   },
+  "build": {
+    "appId": "FOO.org.codaprotocol.prowallet",
```

Comment:
this needs to be a real Apple Team ID if we want to sign our application. Since we don't have a team id yet, I'm sticking FOO here.